src/plugins/vis_type_vega/public/vega_view/vega_map_view/map_service_settings.ts

```diff
@@ -36,11 +38,15 @@ export class MapServiceSettings {
     });
   }
 
+  public get hasUserConfiguredTmsLayer() {
+    return hasUserConfiguredTmsService(this.config);
+  }
+
   public get defaultTmsLayer() {
     if (this.hasUserConfiguredTmsLayer) {
       return TmsTileLayers.userConfigured;
     }
-    return this.uiSettingsValues.isDarkMode ? TmsTileLayers.dark : TmsTileLayers.desaturated;
+    return this.isDarkMode ? TmsTileLayers.dark : TmsTileLayers.desaturated;
```

Comment:
@thomasneirynck which TSM service should be used by default?  I see Map plugin uses `road_map` but in previous version of Vega it looks more like `road_map_desaturated`